client/src/api/Card.js

```diff
@@ -0,0 +1,51 @@
+import axios from 'axios';
+
+// post card
+export const addCardByColumnId = async (columnId, formData) => {
+  try {
+    const config = {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    };
+    const body = JSON.stringify(formData);
+
+    const res = await axios.post(`/api/cards/create/${columnId}`, body, config);
+
+    return {data: res.data, error: false};
+  } catch (err) {
+    return {data: [], error: true};
+  }
+};
+
+// Get card by Id
+export const getCardById = async (card) => {
+  try {
+    // eslint-disable-next-line no-param-reassign, no-underscore-dangle
+    const res = await axios.get(`/api/cards/show/${card._id}`);
+
+    return {data: res.data, loading: false, error: false};
+  } catch (err) {
+    return {data: [], loading: false, error: true};
+  }
+};
+
+// Update Card by Id
+export const updateCard = async (card) => {
+  try {
+    const config = {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    };
+    const body = JSON.stringify(card);
+    // eslint-disable-next-line no-param-reassign, no-underscore-dangle
+    const url = `/api/cards/detail/update/${card._id}`;
```

Comment:
once this is all merged we can go back and simplify some of these routes because they're way too specific